### DISCUSSION TITLE
Need to listen on AR Mode activated event

### DISCUSSION THREAD

**Discussion by GregoryFrance** [google/model-viewer#4378] [Ideas]
I didn't find a way to listen when AR Mode is activated

**Comment by diegoteran** [comment]
Hi, do you have a link to a demo?
Do you mean listening to a model viewer event?

**Comment by GregoryFrance** [comment]
Hello,
I don't have any demo.
In my user case, I would like to log event every time a user enter in AR Mode, to check  functionality popularity. 
I Imagine a 'enter-ar' or 'ar-activated' event...

Exemple:

document.querySelector('#mymodelviewerobject').addEventListener('enter-ar', (event) => {
      mylocalmethod();
    });

**Reply by elalish** [reply]
Try the `ar-status` event: https://modelviewer.dev/docs/index.html#entrydocs-augmentedreality-events-arStatus
